source/docs/software/pathplanning/robot-characterization/characterization-routine.rst

```diff
@@ -18,6 +18,31 @@ If the tool does not seem to be successfully connecting, try rebooting the robot
 .. image:: images/connected-to-robot.png
    :alt: Connected to the robot
 
+
+Project Parameters
+------------------
+
+.. image:: images/projectparameters.png
+   :alt: Project Parameters section of the Logger window.
+
+:guilabel:`Mechanism` controls what data is sampled and how gravity is expected to affect the analysis.
+
+:guilabel:`Unit Type` is the units you are using and the units that SysID will export the gains in.
+
+:guilabel:`Units Per Rotation` defines anything that affects the change from rotations of the encoder shaft to the units selected above.  As an example say you are using a TalonFX motor controller with its integrated encoder, the KOP chassis, and units of meters.  The encoder is inside the motor so we have to account for the gearing of the gearbox which is 10.71:1.  We also have to take into account how our wheel will change the distance we have traveled per rotation.  The standard chassis has 6" (0.1524 meters) diameter wheels, so to get the circumference we need to multiply by Pi.  The calculation looks like:
+
+.. math:: Units Per Rotation = (1 / 10.71) \cdot (0.1524 \cdot \pi)
+
+Voltage Parameters
+------------------
+
+.. image:: images/voltageparameters.png
+   :alt: Voltage Parameters section of the Logger window.
+
+:guilabel:`Quasistatic Ramp Rate` controls how quickly the voltage will ramp up during the quasistatic tests.  The goal here is to get the voltage ramped up almost to full speed.  If the amount of space you have to run the robot is small you might need to slightly increase this ramp rate.
```

Comment:
You don't actually need to sample all the speeds. You just need enough to get a trendline out of it.